myhdl/_Cosimulation.py

```diff
@@ -137,7 +137,10 @@ def _get(self):
                     if next >= (1 << (s._nrbits-1)):
                         next |= (-1 << s._nrbits)
             except ValueError:
-                next = intbv(0)
+                if v in 'xX':
+                    next = s._init
```

Comment:
Why `x` instead of `z`?  If `z` was used for tri-stated value in cosimulation, the first approach would probably work.  I think it is an issue because all the Verilog signals are 'x' at T0, it tries to sync everything to None.  If 'z' is used instead then the None could be used.  

I am worried that for non-tristate signals, in the past an 'x' would goto 0 and now it will go back to the initial value (grant it 0 is a common init value).  This might change the behavior or existing code.